modules/distr-tracing-otel-config-collector.adoc

```diff
@@ -365,3 +365,56 @@ The Logging exporter prints data to the standard output.
           exporters: [logging]
 ----
 <1> Verbosity of the logging export: `detailed` or `normal` or `basic`. When set to `detailed`, pipeline data is verbosely logged. Defaults to `normal`.
+
+[id="extensions_{context}"]
+=== Extensions
+
+[id="basicauth-extension_{context}"]
+==== BASICAUTH extension
```

Comment:
We do have `OTLP exporter` and `OTLP HTTP exporter` because OTLP and HTTP are abbreviations, but  I think we don't need to capitalize the whole `BASICAUTH`:
```suggestion
==== BasicAuth extension
```